### PR TITLE
Add minor features

### PR DESCRIPTION
**Pull request by samgnaniah**
Adds minor features to About this Release page.

### CODE REVIEW

File: en/docs/get-started/about-this-release.md [hunk 1]
```diff
@@ -91,16 +96,23 @@ The Micro Integrator dashboard can connect to multiple Micro Integrator server i
...
+     Users can use OIDC based Single Sign-On to log in to the WSO2 Micro Integrator Monitoring Dashboard.
 
 ## **Streaming Integrator Tooling**
 
 The Streaming Integrator Tooling is a developer tool to develop Siddhi applications and simulate events for testing purposes.
 
 It is now available to download from [here](https://github.com/wso2/streaming-integrator-tooling/releases/).
 
-<! --- ### New features --->
+### New features
```
Comment (AnuGayan) [inline en/docs/get-started/about-this-release.md:111]: Shall we add this under Streaming Integrator section..

File: en/docs/get-started/about-this-release.md [hunk 2]
```diff
@@ -91,16 +96,23 @@ The Micro Integrator dashboard can connect to multiple Micro Integrator server i
...
+
+- **Add support for SMB and Webdav protocols**
```
Comment (Mariangela) [inline en/docs/get-started/about-this-release.md:113]:
```suggestion
- **Add support for SMB and WebDAV protocols**
```

File: en/docs/get-started/about-this-release.md [hunk 3]
```diff
@@ -91,16 +96,23 @@ The Micro Integrator dashboard can connect to multiple Micro Integrator server i
...
+
+  This feature adds support to read from or write to files opened via SMB and Webdav protocols.
```
Comment (Mariangela) [inline en/docs/get-started/about-this-release.md:115]:
```suggestion
  This feature adds support to read from or write to files opened via SMB and WebDAV protocols.
```

**Review by Nirothipan — APPROVED** [review]
MI part is reviewed

File: en/docs/get-started/about-this-release.md (hunk 3 above)
Comment (samgnaniah) [reply en/docs/get-started/about-this-release.md]: Fixed.

File: en/docs/get-started/about-this-release.md (hunk 2 above)
Comment (samgnaniah) [reply en/docs/get-started/about-this-release.md]: Fixed.

File: en/docs/get-started/about-this-release.md (hunk 1 above)
Comment (samgnaniah) [reply en/docs/get-started/about-this-release.md]: Done.

**Pull request merged**